Source/FuseJS/Bundle.uno

```diff
@@ -53,14 +53,15 @@ namespace FuseJS
 			}
 			```
 		*/
-		static Future<byte[]> GetBuffer(object[] args)
+		public static Future<byte[]> ReadBuffer(object[] args)
```

Comment:
Don't make these public for testing, make them `internal` and use InternalsVisibleTo. The person that needs these visible for public use can make them public then.